docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -20,14 +20,15 @@ session.set_providers(['OpenVINOExecutionProvider'], [{Key1 : Value1, Key2 : Val
 *Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization*
 
 ### C/C++ API
-All the options (key-value pairs) need to be concantenated into a string as shown below and passed to OrtSessionOptionsAppendExecutionProviderEx_OpenVINO() API as shown below:-
+All the options shown below are passed to SessionOptionsAppendExecutionProvider_OpenVINO() API and populated in the struct OrtOpenVINOProviderOptions in an example shown below:-
 
 ```
-std::string settings_str;
-settings_str.append("Key1|Value1\n");
-settings_str.append("Key2|Value2\n");
-settings_str.append("Key3|Value3\n");
-Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProviderEx_OpenVINO(sf, settings_str.c_str()));
+OrtOpenVINOProviderOptions options;
+options.device_type = "CPU_FP32";
+options.enable_vpu_fast_compile = 0;
+options.device_id = "";
+options.num_of_threads = 8;
+SessionOptionsAppendExecutionProvider_OpenVINO(session_options, &options);
```

Comment:
@sfatimar 
sahar, I think SessionOptionsAppendExecutionProvider_OpenVINO(session_options, &options);    won't get compiled.

It should be sf.AppendExecutionProvider_OpenVINO(options);
                             (or)
                    session_options.AppendExecutionProvider_OpenVINO(options);

Reply:
it will work, it is a C api from onnxruntime_c_api.h

from onnxruntime_cxx_api.h is a C++ api ... 
session_options.AppendExecutionProvider_OpenVINO(options);

Reply:
@sfatimar  
oh I see. so shouldn't we be adding two separate headings how to use C API and how to use c++ API?